### PR TITLE
Code cleanup on [bancor-pool-token-underlying-balance] strategies

### PR DESCRIPTION
**Pull request by TudorSante**
Fixes # .

Changes proposed in this pull request:
- Performed some code cleanup on [xrook-balance-of-underlying-weighted] and [bancor-pool-token-underlying-balance] strategies

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/index.ts [hunk 1]
```diff
@@ -540,7 +538,6 @@ const strategies = {
   snowswap,
   meebitsdao,
   'crucible-erc20-balance-of': crucibleERC20BalanceOf,
-  'crucible-erc20-token-and-lp-weighted': crucibleERC20TokenAndLpWeighted,
```
Comment (ChaituVR) [inline src/strategies/index.ts:543]: Make sure this file is not changed

File: src/strategies/bancor-pool-token-underlying-balance/index.ts [hunk 2]
```diff
@@ -54,6 +54,9 @@ export async function strategy(
   ).then((res) => parseFloat(formatUnits(res, options.decimals)));
 
   return Object.fromEntries(
-    Object.entries(scores).map((res: any) => [res[0], res[1] * underlyingValue])
+    Object.entries(scores).map((res: any) => [
+      res[0], 
+      res[1] * underlyingValue
+    ])
```
[the remark this answers is not in the document]
Comment (TudorSante) [reply src/strategies/bancor-pool-token-underlying-balance/index.ts:60]: Yea, received some lint fixes on a strategy and wanted to add them everywhere. Also I'm getting familiar to using Git, so sorry if some things are off-track. Gonna implement the above asap

File: src/strategies/bancor-pool-token-underlying-balance/examples.json [hunk 3]
```diff
@@ -16,6 +16,6 @@
       "0x857eb0eb2572f7092c417cd386ba82e45eba9b8a",
       "0x98625957c6b96eeecd040e5bd26bd42100d124b2"
     ],
-    "snapshot": 15191222
+    "snapshot": "latest"
```
[the remark this answers is not in the document]
Comment (TudorSante) [reply src/strategies/bancor-pool-token-underlying-balance/examples.json]: Done

File: src/strategies/index.ts (hunk 1 above)
Comment (TudorSante) [reply src/strategies/index.ts]: Hopefully this problem was solved

**Comment by ChaituVR** [comment]
@TudorSante These changes are just lint related and our lint runner already fixed these, and these changes make it bad, so closing it for now,  feel free to reopen if there is a change not related to lint :)

**Pull request closed without merging**